sites/partners/pages/index.tsx

```diff
@@ -137,7 +145,23 @@ export default function ListingsList() {
         <article className="flex-row flex-wrap relative max-w-screen-xl mx-auto py-8 px-4">
           <div className="ag-theme-alpine ag-theme-bloom">
             <div className="flex justify-between">
-              <div className="w-56"></div>
+              <div className="w-56">
+                <Select
+                  defaultValue={options[0]}
+                  isMulti
```

Comment:
Is this what lets you select multiple neighborhoods? We'll need to figure out how to do that on the backend.

Reply:
Yes, these props allow the multi-select and I just hardcoded the first option to show selected.  Mostly was just playing around with the various props.